IPython/lib/tests/test_deepreload.py

```diff
@@ -17,11 +17,10 @@ def test_deepreload():
     "Test that dreload does deep reloads and skips excluded modules."
     with TemporaryDirectory() as tmpdir:
         with prepended_to_syspath(tmpdir):
-            tmpdirpath = Path(tmpdir)
-            with open(tmpdirpath / "A.py", "w") as f:
-                f.write("class Object(object):\n    pass\n")
-            with open(tmpdirpath / "B.py", "w") as f:
-                f.write("import A\n")
+            tmpdirpathA = Path(tmpdir + "A.py")
+            tmpdirpathB = Path(tmpdir + "B.py")
+            tmpdirpathA.write_text("class Object(object):\n    pass\n")
+            tmpdirpathB.write_text("import A\n")
```

Comment:
That is incorrect and test are failing. Please check your work, I belive you want / instead fo +.

Reply:
Exactly! Thanks for the reminder. Will do ASAP.